charts/oes/values.yaml

```diff
@@ -427,6 +466,16 @@ opa:
     tag: latest
     pullPolicy: IfNotPresent
 
+#####################################################
+## Forwarder/Controller configuration
+#####################################################
+forwarder:
+  agentServiceType: LoadBalancer
+  externalName: grpc.controller.opsmx.com
```

Comment:
Let's add comments for each of these things explaining why we need it and when to change the default values.

Reply:
Done